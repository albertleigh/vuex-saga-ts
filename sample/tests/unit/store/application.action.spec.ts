import { SagaActions } from '@/store/application/actions';

describe('Application actions',()=>{

    describe('act creators',()=>{

        it('start',()=>{
            expect(''+SagaActions.start).toMatchSnapshot('application_saga_act_start_name');
            expect(SagaActions.start()).toMatchSnapshot('application_saga_act_start');
        })

    })

})
