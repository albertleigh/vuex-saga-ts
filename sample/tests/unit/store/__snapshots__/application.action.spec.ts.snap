// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application actions act creators start: application_saga_act_start 1`] = `
Object {
  "payload": Object {},
  "type": "APPLICATION_START",
}
`;

exports[`Application actions act creators start: application_saga_act_start_name 1`] = `"APPLICATION_START"`;
